server/server.ts

```diff
@@ -397,5 +395,52 @@ app.post("/webhook", async(req, res) =>{
       // Return a 200 response to acknowledge receipt of the event
       res.json({received: true});
 })
+
+const propToString = prop => obj =>
+    ({...obj, [prop]: obj[prop].toString()})
+    
+app.get('/api/plays', async (req, res) => {
+    try {
+        const querystring = `
+            SELECT id, playname title, playdescription description, image_url
+            FROM plays
+            WHERE active=true;`
+        const data = await pool.query(querystring)
+        const plays = data.rows.map(propToString('id'))
+        res.json(plays)
+    }
+    catch (err) {
+        console.error(err.message);
+    }
+});
+
+// remove $ and parse to float
+const parseMoneyString = (s: string) => Number.parseFloat(s.slice(1))
+const toTicket = (row): Ticket => ({
+    ...row,
+    playid: row.playid.toString(),
+    ticket_price: parseMoneyString(row.ticket_price),
+    concession_price: parseMoneyString(row.concession_price),
+    eventdate: dayMonthDate(row.eventdate),
+    starttime: militaryToCivilian(row.starttime),
+})
+
+app.get('/api/tickets', async (req, res) => {
+    try {
+        const qs = `SELECT sh.id eventid, playid, playname event_title, eventdate, starttime, availableseats, tt.name admission_type, price ticket_price, concessions concession_price
```

Comment:
looks like a few commas are missing here?

Reply:
I left out the 'as' keywords here, but the goal was to get column names to align with the types I had defined.